blog/updating_our_tracking_prevalence_metrics.md

```diff
@@ -0,0 +1,98 @@
+title: Updating our tracking prevalence metrics
+subtitle: Making metrics that make more sense.
+author: privacy team
+type: article
+publish: True
+date: 2019-02-22
+tags: blog, update
+header_img: blog/updating-prevalence-metrics/header.jpg
++++
+
+This month we updated the site with data from over 800 million page loads during January. In this
+data release we have also made a change to one of the main figures that we publish - `site reach`.
+Metrics are as good as their ability to capture a simplified version of reality. The
+main motivation for redefining `site reach` attempts to achieve exactly that - simplify the
+understanding of prevalence.
+
+The site reach stat was conceived as a measure of how many different sites a tracker has some
+presence. In contrast to `reach` - the proportion of pages loading a tracker - it shows how spread
+around the web a tracker is.
+
+We define a new site reach metric as:
+> The number of sites in the top 10,000 which have this tracker on more than 1% of page loads.
+
+The relationship between reach and site reach paints an interesting picture of prevalence:
+
+![Reach vs Site Reach, Jan 2019](../static/img/blog/updating-prevalence-metrics/reach-vs-sitereach.svg)
+<p class="img-caption">Figure 1: Reach vs Site Reach, Jan 2019 (source: whotracks.me)</p>
+
+
+- High reach and high site reach - Ubiquitous presence across both popular and less popular sites;
+A common example of that would be [Google Analytics](../trackers/google_analytics.html).
+- High reach and low site reach - Present primarily on few popular, high-traffic sites;
+One such example would be [Wikimedia](../trackers/wikimedia.org.html), which is due to
```

Comment:
"which is due to Wikipedia's popularity is loaded very often", should this be "which, due to Wikipedia's [...]"?